docs/man/man1/tkn-clustertask-delete.1

```diff
@@ -0,0 +1,66 @@
+.TH "TKN\-CLUSTERTASK\-DELETE" "1" "Aug 2019" "Auto generated by spf13/cobra" "" 
```

Comment:
This document and ```docs/man/man1/tkn-clustertask.1 ``` is not related this PR, but I got diff when run ```make man``` and ```make docs``` so I included these.